src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+    protected String value;
+    protected boolean isPrivate;
+
+    public Contact(boolean isPrivate) {
```

Comment:
It is better to set value in this superclass since all the subclasses set their value in the same way. If you have questions related to this comment, feel free to email me or ask me next week.